airbyte-webapp/src/locales/en.json

```diff
@@ -510,6 +518,9 @@
   "settings.account": "Account",
   "settings.accountSettings.updateEmailSuccess": "Email updated",
   "settings.cookiePreferences": "Cookie Preferences",
+  "settings.defaultDataResidency": "Default Data residency",
```

Comment:
Suggestion: Should we maybe rename this just "Data Residency"? "Default Data Residency" feels like weirdly specific name for the whole page (since it's basically the name of the setting itself). So I feel we could go with `Data residency` and potentially be more flexible adding other related information there later.

Reply:
Absolutely, that makes sense

Reply:
Fixed, and adjusted a few variable names too: https://github.com/airbytehq/airbyte/pull/18611/commits/a62ff9b34c72c8344b61e0e555654e72b241f3ce

Reply:
I think the strnig though still says "Default Data Residency"? :D

Reply:
Ah yeah fair 😄 I was thinking of changing just the URL. But it makes sense for the menu item to also be "Data Residency"